ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -280,7 +280,7 @@ library
     , time
     , transformers
     , typed-protocols              ^>=0.1.1
-    , vector                       ^>=0.12
+    , vector                       ^>=0.12    || ^>=0.13
```

Comment:
Hmm, maybe we want to actually drop 0.12 support? Or is it still required upstream? As we talked about recently, we might otherwise accidentally drop support for (ie make compilation fail with) 0.12 without noticing it.